src/assets/sass/v2/blocks/_panel.scss

```diff
@@ -0,0 +1,12 @@
+.panel-body-title--white-text {
```

Comment:
Since we are diverging from the design of the standard component, we could create a new component of our own that extends the original but adds a modifier to allow smaller text in the body section:
```
.panel {
  @extends .gov-uk-panel;
}
.panel--confirmation {
  @extends .gov-uk-panel--confirmation;
}
.panel__title {
  @extends .gov-uk-panel__title;
}
.panel__body {
  @extends .gov-uk-panel__body;
}
.panel__body--small {
  font-size: 1rem; 
}
```